### DISCUSSION TITLE
Ubuntu 24.04 Support?

### DISCUSSION THREAD

**Discussion by Droning1355** [ConSol/omd#190] [General]
Ist ein Support für Ubuntu 24.04 geplant

**Comment by sni** [comment]
we usually add new platforms when we need them on customer deployments. So there are no direct plans to do so right now, but sooner or later it will be added. I'd say you can expect it within the next 6months or so.

**Reply by Droning1355** [reply]
thanks for your work!! and the fast reply 👍
